### ISSUE TITLE
provide the api to list all exist data key from redis db

### ISSUE THREAD

**Issue by PenHsuanWang** [PenHsuanWang/stock-analyzer-backend#3]
expose the serving api endpoint for list all exist data.

**Comment by PenHsuanWang** [comment]
perform the result

example of usage
```python
def get_all_data_keys():
    endpoint = "/stock_data/get_all_keys"
    response = requests.get(f"{BASE_URL}{endpoint}")
    data = response.json()
    return data.get("keys", [])

if __name__ == "__main__":
    # Fetch all data keys from the server
    all_keys = get_all_data_keys()
    print(f"Available data keys in the server: {all_keys}")

```
Available data keys in the server: ['stock_data:GOOGL:2023-01-01:2023-10-10', 'stock_data:AAPL:2023-01-01:2023-10-10', 'stock_data:TSM:2023-01-01:2023-10-10']
```

**Comment by PenHsuanWang** [comment]
fixed with commit 3c8dcdf
